crates/zksync/core/src/vm/runner.rs

```diff
@@ -174,6 +174,7 @@ pub fn call<DB, E>(
     call: &CallInputs,
     ecx: &mut EvmContext<DB>,
     mut ccx: CheatcodeTracerContext,
+    factory_deps: Vec<Vec<u8>>,
```

Comment:
Let's keep the signature consistent with `create` and have `factory_deps` directly before `ecx`